src/UICircularProgressRing/UICircularProgressRing.swift

```diff
@@ -135,6 +135,20 @@ final public class UICircularProgressRing: UICircularRing {
         didSet { ringLayer.maxValue = maxValue }
     }
 
+    /**
+     A variable to determine if the value label will count down to zero.
+
+     ## Important ##
+     Default = false
+
+	## Author
+	Jonah Ollman
+	*/
+
+	@IBInspectable public var isReverse: Bool = false {
```

Comment:
I like the idea of this feature, however don't think it should be a property.
We should do something like

`startProgress(to: 100, inReverse: Bool)`